ApollosProject/apollos-apps
# Moves HorizontalContentSeriesFeedConnected to the ui-connected package

## DESCRIPTION
<img src="https://user-images.githubusercontent.com/2053547/73972418-d15a6500-48ee-11ea-84be-d401b6e1d074.gif" width="30%" />

### What does this PR do, or why is it needed?
This PR moves the formerly named `HorizontalContentFeed` to the `ui-connected` package and renames it to `HorizontalContentSeriesFeedConnected`. It also does some mild clean up and refactoring to make it follow already established paradigms and to enabled placeholder loading states. Please review this closely and test on both platforms. 

### What design trade-offs/decisions were made?
N/A
### How do I test this PR?
open app and tap on a series or series child. The feed at the bottom should show up and display a loading state. 

### What automated tests did you write?
No change in test coverage.

## TODO:

- [x] Upload GIF(s) of iOS

## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
